### PR TITLE
Update admin login pages

### PR DESCRIPTION
**Pull request by luigifab**
### Description

This PR update admin login page, admin forgot password, admin reset password, for default and openmage themes.

login

forgot

reset

login-om

forgot-om

reset-om

### Contribution checklist
- [x] Pull request has a meaningful description of its purpose
- [x] All commits are accompanied by meaningful commit messages
- [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ADDISON74** [comment]
As a first suggestion we should remove from the code all lines related to Internet Explorer. There are still template files with these lines.

```
<!--[if IE]> <link rel="stylesheet" href="<?php echo $this->getSkinUrl('iestyles.css') ?>" type="text/css" media="all" /> <![endif]-->
<!--[if lt IE 7]> <link rel="stylesheet" href="<?php echo $this->getSkinUrl('below_ie7.css') ?>" type="text/css" media="all" /> <![endif]-->
<!--[if IE 7]> <link rel="stylesheet" href="<?php echo $this->getSkinUrl('ie7.css') ?>" type="text/css" media="all" /> <![endif]-->
```
As a second suggestion this time a subjective one I prefer more than "OpenMage LTS" something like "Powered by OpenMage". I'd loved to know more opinions related to this phrase.

**Comment by luigifab** [comment]
The removal of IE is in #1073.

**Comment by fballiano** [comment]
There's already a PR involving this screen (it's about the logo actually) and I think there are going to be some conflicts :-(

**Comment by luigifab** [comment]
No conflict with #1599 (I change only phtml here).
And @ADDISON74, yes, **Powered by OpenMage** is better than **OpenMage (LTS)**.

**Review by fballiano — DISMISSED** [review]

**Comment by luigifab** [comment]
With powered:
image

**Comment by luigifab** [comment]
When this PR will be merged in 1.9.4.x, a conflict will appear with 20.0 due to `<!--[if IE]>` removal.

File: app/design/adminhtml/default/openmage/template/login.phtml [hunk 1]
```diff
@@ -31,48 +31,49 @@
...
+                    <?php echo $this->getChildHtml('form.additional.info') ?>
                     <div class="clear"></div>
                     <div class="form-buttons">
                         <a class="left" href="<?php echo Mage::helper('adminhtml')->getUrl('adminhtml/index/forgotpassword', array('_nosecret' => true)) ?>"><?php echo Mage::helper('adminhtml')->__('Forgot your password?') ?></a>
-                        <input type="submit" class="form-button" value="<?php echo Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" title="<?php echo Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" /></div>
+                        <input type="submit" class="form-button" value="<?php echo Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" title="<?php echo Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" />
+                    </div>
                 </div>
-                <p class="legal"><?php echo Mage::helper('adminhtml')->__('OpenMage LTS') ?></p>
+                <p class="legal"><?php echo Mage::helper('adminhtml')->__('Powered by OpenMage') ?></p>
```
Comment (ADDISON74) [inline app/design/adminhtml/default/openmage/template/login.phtml:70]: You should decide if it is "OpenMage" or "OpenMage LTS". Having already two rows based on "OpenMage LTS" I would change this line to "Powered by OpenMage LTS". If you add LTS don't forget to change in translation this record.

Comment (luigifab) [reply app/design/adminhtml/default/openmage/template/login.phtml]: Oké!

File: app/design/adminhtml/default/default/template/login.phtml [hunk 2]
```diff
@@ -29,52 +29,52 @@
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <meta name="robots" content="noindex, nofollow" />
-    <title><?php echo Mage::helper('adminhtml')->__('Log into OpenMage Admin Page') ?></title>
+    <title><?php echo Mage::helper('adminhtml')->__('Log into OpenMage LTS Admin Page') ?></title>
```
Comment (ADDISON74) [inline app/design/adminhtml/default/default/template/login.phtml:32]: I would keep OpenMage only not OpenMage LTS.

File: app/design/adminhtml/default/openmage/template/login.phtml (hunk 1 above)
Comment (ADDISON74) [reply app/design/adminhtml/default/openmage/template/login.phtml]: From what I checked you opted for OpenMage LTS. It is good to keep all changes the same. I was less interested in whether it was with or without LTS.

**Comment by fballiano** [comment]
why we move from "openmage" to "openmage lts"? like, to me the openmage name is a wonderful name, "magento lts" or "openmage lts" are just watering it down, at least in my opinion

**Comment by luigifab** [comment]
I prefer "OpenMage" only, but I will heart feeling of some people if I do that, if I remember correctly.

**Comment by kiatng** [comment]
+1 for OpenMage. And informally, OM.

**Comment by ADDISON74** [comment]
There are a lot of LTS in this PR not only "Powered by OpenMage **LTS**". For example, ```<title><?php echo Mage::helper('adminhtml')->__('Log into OpenMage LTS Admin Page') ?></title>```.

Once all are removed I will approve it.

**Review by fballiano — APPROVED** [review]

**Review by kiatng — APPROVED** [review]
Tested, looking good.

**Review by ADDISON74 — APPROVED** [review]

**Comment by Flyingmana** [comment]
was there somewhere an explaination why we changed this to multiple entries?
```
<script type="text/javascript" src="<?php echo $this->getJsUrl(); ?>index.php/x.js?f=prototype/prototype.js,prototype/validation.js,mage/adminhtml/events.js,mage/adminhtml/form.js,scriptaculous/effects.js"></script>
```

I dont object to it, I just would expect to have it mentioned

**Comment by fballiano** [comment]
I agree with this PR, there's no real reason anymore to merge the js via php, with http2 merging is almost completely useless and lighthouse (as far as i know) doesn't suggest merging anymore. so with this PR we remove some server side computation ;-)

**Comment by justinbeaty** [comment]
@Flyingmana it was mentioned in #1622

**Pull request merged**